manifests/init.pp

```diff
@@ -101,4 +101,13 @@
     environment => concat([ "VENV_PATH=${venv_path}" ], $environment),
     refreshonly => true,
   }
+
+  # Used in letsencrypt::certonly Exec["letsencrypt certonly ${title}"]
+  file { '/usr/local/sbin/letsencrypt-domain-validation':
```

Comment:
The script is only used into `letsencrypt::certonly`, why is it created here and not with the same resource that need it ?

Reply:
I've decided against moving it into `letsencrypt::certonly` because i would have to use `defined()` which might lead to unnecessary performance degradation on systems with a lot of `letsencrypt::certonly`'s. Also `letsencrypt::certonly` requires `Class[letsencrypt]` to be used, so this should work flawlessly.

Reply:
isn't `/usr/local/sbin/` and it should be in `usr/local/bin` instead?

Reply:
Per definition, scripts which are root only/system administration script are placed in */sbin. Thats the reason I chose this location.